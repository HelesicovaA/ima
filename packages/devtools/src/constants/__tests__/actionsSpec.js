import Actions from '../actions';

describe('actions constant', () => {
  it('should match snapshot', () => {
    expect(Actions).toMatchSnapshot();
  });
});
