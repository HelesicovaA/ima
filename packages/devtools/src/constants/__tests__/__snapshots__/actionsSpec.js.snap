// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions constant should match snapshot 1`] = `
Object {
  "ALIVE": "ima:devtool:alive",
  "DEAD": "ima:devtool:dead",
  "DETECTING": "ima:devtool:detecting",
  "MESSAGE": "ima:devtool:message",
  "POPUP": "ima:devtool:popup",
  "RELOADING": "ima:devtool:reloading",
  "SETTINGS": "ima:devtool:settings",
  "UNSUPPORTED": "ima:devtool:unsupported",
}
`;
